src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const funcObj = function(x) {
```

Comment:
```suggestion
  return function(x) {
```

Reply:
fixed
got it
thank you for your advises